### PR TITLE
IGVF-2162-fix-bulk-crispr-audits

### PR DESCRIPTION
**Pull request by ian-whaling**

### CODE REVIEW

File: src/igvfd/schemas/changelogs/auxiliary_set.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ### Minor changes since schema version 11
 
+* Adjust `file_set_type` enum list to remove `variant sequencing`.
```
Comment (jenjou) [inline src/igvfd/schemas/changelogs/auxiliary_set.md:5]: Should be in a new section for schema ver 12

**Review by jenjou — CHANGES_REQUESTED** [review]
See comments

File: src/igvfd/tests/test_audit_measurement_set.py [hunk 2]
```diff
@@ -732,41 +732,6 @@ def test_audit_unexpected_virtual_sample(
...
-        for error in res.json['audit'].get('WARNING', [])
-    )
-    testapp.patch_json(
-        measurement_set_no_files['@id'],
-        {
-            'assay_term': assay_term_crispr['@id']
-        }
-    )
-    res = testapp.get(measurement_set_no_files['@id'] + '@@audit')
-    assert all(
```
[the remark this answers is not in the document]
Comment (ian-whaling) [reply src/igvfd/tests/test_audit_measurement_set.py:753]: There already exists a test for testing just the missing files portion of the audit in test_audit_file_set.py with test_audit_missing_files.

File: src/igvfd/schemas/changelogs/auxiliary_set.md (hunk 1 above)
Comment (ian-whaling) [reply src/igvfd/schemas/changelogs/auxiliary_set.md:5]: fixed, my mistake.

File: src/igvfd/tests/test_audit_measurement_set.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (ian-whaling) [reply src/igvfd/tests/test_audit_measurement_set.py:753]: I also cannot get coveralls to load

**Review by jenjou — DISMISSED** [review]
Needs rebase due to mappings, otherwise looks good.

**Pull request merged**